src/script/page/preferences/devices/DevicesPreferences.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {useState} from 'react';
+import {ClientEntity} from 'src/script/client/ClientEntity';
+import {ClientState} from '../../../client/ClientState';
+import {UserState} from '../../../user/UserState';
+import {ConversationState} from '../../../conversation/ConversationState';
+import {container} from 'tsyringe';
+import {registerStaticReactComponent, useKoSubscribableChildren} from '../../../util/ComponentUtil';
+import {QualifiedId} from '@wireapp/api-client/src/user';
+import {t} from 'Util/LocalizerUtil';
+import VerifiedIcon from 'Components/VerifiedIcon';
+import Icon from 'Components/Icon';
+import {CryptographyRepository} from 'src/script/cryptography/CryptographyRepository';
+import {useFadingScrollbar} from '../../../ui/fadingScrollbar';
+import useEffectRef from 'Util/useEffectRef';
+import DetailedDevice from './components/DetailedDevice';
+import DeviceDetailsPreferences from './DeviceDetailsPreferences';
+import {Conversation} from '../../../entity/Conversation';
+import {FormattedId} from './components/FormattedId';
+
+interface DevicesPreferencesProps {
+  clientState: ClientState;
+  conversationState: ConversationState;
+  cryptographyRepository: CryptographyRepository;
+  removeDevice: (device: ClientEntity) => Promise<void>;
+  resetSession: (userId: QualifiedId, device: ClientEntity, conversation: Conversation) => Promise<void>;
+  userState: UserState;
+  verifyDevice: (userId: QualifiedId, device: ClientEntity, isVerified: boolean) => void;
+}
+
+const Device: React.FC<{
+  device: ClientEntity;
+  isSSO: boolean;
+  onRemove: (device: ClientEntity) => void;
+  onSelect: (device: ClientEntity) => void;
+}> = ({device, isSSO, onSelect, onRemove}) => {
+  const {isVerified} = useKoSubscribableChildren(device.meta, ['isVerified']);
+
+  return (
+    <div className="preferences-devices-card" onClick={() => onSelect(device)}>
```

Comment:
clickable `div` should have `role="button"` attribute, `tabIntex={0}` attribute and should handle the `onKeyPress` event (or another key event) to allow doing the same as `onClick` does for Enter and Space keys